config/config/sc-config.yaml

```diff
@@ -129,6 +129,14 @@ grafana:
     #     timeout: 600
   user:
     enabled: true
+    sidecar:
```

Comment:
this is not needed, as the values should be templated from the grafana.ops key

Reply:
It looks to me as if "ops" and "user" have different values if im not mistaken?:

```
resources: {{- toYaml .Values.grafana.user.sidecar.resources | nindent 6 }}

resources: {{- toYaml .Values.grafana.ops.sidecar.resources | nindent 6 }}
```

Reply:
the values are templated from the ops section (see `<<: *grafanadefaults` below)

Reply:
Thanks, took your directions and it worked out fine!